test-cases/README.md

```diff
@@ -167,45 +158,88 @@ This command will read all test results from Jira, and upload them to Polarion.
 JIRA_USERNAME=ju JIRA_PASSWORD=jp POLARION_USERNAME=pu POLARION_PASSWORD=pp ./tools.sh polarion testrun --epic INTLY-5390
 ```
 
-## Tags
+## Test Case Metadata
 
-Tags can be added for each test case inside the metadata section in the header. Tags can be defined globally
-or specifically for each variant.
+### Tags
 
-Supported tags:
+Tags are used to define specific behaviors for the test case.
 
-- `draft` Indicates that this test case is uncompleted and should not be executed.
 - `automated` Indicates that this test case is already automated and that it doesn't need to be executed manually
-- `obsolete` Indicates that this test case is obsolete and that it must be updated as soon as possible
+- `per-build` Indicates that this test case should be executed on each build even if passed in the previous one
+- `per-release` Automatically generated if the `targets` field is set, and it indicates that the tests will be executed on each release
 
-## Prettier
-
-> All of the test cases must be prettify before being committed
+E.g.:
 
-You can prettify the test cases using the command line:
+```
+---
+tags:
+  - per-build
+---
 
-```bash
-npm run prettier
+# Z00 - Verify
 ```
 
-Or using the VS Code extension: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
+### Components
 
-## Export to CSV
+The component field is used to decide which tests should be added in the next release.
 
-Run the `./tools.sh export csv` script to export all test cases in a CSV table:
+- `product-3scale` 3Scale minor or major product upgrades
+- `product-amq` AMQ minor or major product upgrades
+- `product-fuse` Fuse minor or major product upgrades
+- `product-codeready` CodeReady minor or major product upgrades
+- `product-sso` SSO minor or major product upgrades
+- `product-ups` UPS minor or major product upgrades
+- `product-data-sync` Data Sync minor or major product upgrades
+- `monitoring` Monitoring stack changes or upgrade
+- `openshift` Special component using for testing OpenShift upgrades
+
+### Targets
+
+The Targets define against which release the test case would have to be executed the next time. This is useful to include or exclude the test cases form a specific release.
```

Comment:
```suggestion
Targets tag defines against which release the test case is going to be executed next time. This is useful to include or exclude the test cases form a specific release.
```

Reply:
Actually **targets** is a filed not a tag, if I put `Targets tag ...` it will be confusing